### PR TITLE
Manual make update-common and make gen

### PR DESCRIPTION
**Pull request by ericvn**
The automatic update #1725 failed due to make gen failing. This adds that step.

### CODE REVIEW

File: annotation/annotations.gen.go [hunk 1]
```diff
@@ -301,6 +345,7 @@ var (
 		Name: "sidecar.istio.io/inject",
 		Description: "Specifies whether or not an Envoy sidecar should be "+
                         "automatically injected into the workload.",
+		FeatureStatus: Alpha,
```
Comment (howardjohn) [inline annotation/annotations.gen.go:348]: why is this alpha @nmittler

**Review by howardjohn — APPROVED** [review]
approve since its running `make gen`, but this doesn't look right with marking everything alpha?

File: annotation/annotations.gen.go (hunk 1 above)
Comment (nmittler) [reply annotation/annotations.gen.go:348]: Everything defaults to alpha for now. I have a PR asking others to help fill in feature status of all annotations: https://github.com/istio/api/pull/1723

**Review by linsun — APPROVED** [review]

**Pull request merged**